packages/pwa-kit-react-sdk/src/ssr/universal/contexts/index.js

```diff
@@ -24,7 +25,7 @@ const CorrelationIdProvider = ({children, correlationId, resetOnPageChange = tru
     const _correlationIdFn = typeof correlationId === 'function' && correlationId
     const _correlationId = typeof correlationId !== 'function' && correlationId
     if (resetOnPageChange && !_correlationIdFn) {
-        console.warn(
+        logger.warn(
```

Comment:
There's a similar file (components/refresh/index.js) whose `console` is not converted yet: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c8aa0c0525a9e6e3292dc01fdd36d8c80dd26387/packages/pwa-kit-react-sdk/src/ssr/universal/components/refresh/index.jsx#L50

Reply:
And also here https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c8aa0c0525a9e6e3292dc01fdd36d8c80dd26387/packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx#L28-L30